src/AuthenticationMiddleware.php

```diff
@@ -18,10 +18,13 @@
 class AuthenticationMiddleware implements MiddlewareInterface
 {
     /**
-     * @var AuthenticationInterface
+     * @var AuthenticationInterface|\Zend\Expressive\Authentication\AuthenticationInterface
      */
     protected $auth;
 
+    /**
+     * @param AuthenticationInterface|\Zend\Expressive\Authentication\AuthenticationInterface $auth
```

Comment:
Looking at this: why is it a union type, since the parameter type is enforced?

Reply:
Can I revert this changes?

Reply:
IMO yes

Reply:
> IMO yes

Done.